tests/intern.ts

```diff
@@ -56,7 +56,8 @@ export const loaderOptions = {
 		{ name: 'dojo', location: 'node_modules/intern/browser_modules/dojo' },
 		{ name: '@dojo', location: 'node_modules/@dojo' },
 		{ name: 'sinon', location: 'node_modules/sinon/pkg', main: 'sinon' },
-		{ name: 'rxjs', location: 'node_modules/@reactivex/rxjs/dist/amd', main: 'Rx.js' }
+		{ name: 'rxjs', location: 'node_modules/@reactivex/rxjs/dist/amd', main: 'Rx.js' },
```

Comment:
Do we still need rxjs here? Probably not related to this PR, but might as well clean it up since this PR is cleaning up many things...